artemis-distribution/src/main/assembly/dep.xml

```diff
@@ -125,6 +125,12 @@
             <include>org.objenesis:objenesis</include>
             <include>com.typesafe:config</include>
             <include>io.github.classgraph:classgraph</include>
+
+            <include>org.apache.curator:curator-recipes</include>
+            <include>org.apache.curator:curator-client</include>
+            <include>org.apache.curator:curator-framework</include>
```

Comment:
@michaelandrepearce these are the ZK deps

Reply:
Now out of office eh, still need to rebase vs master and include other deps for logging  and force 3.6.1 ZK client version